### PR TITLE
WIP added news index page and made some example news articles

### LINKED ISSUE #16: As a visitor, I want to see a list of all Performant news items
This should be a page with a list of news items in any category (blog posts, announcements, recent projects, etc). Most recent at top (reverse chronologically ordered), with a title, excerpt and link to the main article page. It should live at `/news`

### LINKED ISSUE #17: Per-article news item page
This is a page where a single full news item can be shown, at its own URL. It should include links to share via twitter, facebook, etc.

### PR DESCRIPTION
**Pull request by mkobayashi21**
The articles are examples and the links are WIP to address issue 17
Closes #16 
Closes #17

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -6,12 +6,14 @@ <h1 id="logo"><a href="/"> <img src="/images/performant.png" height="50px" alt="
 			<li><a href="#main" class="scrolly">What We Do</a></li>
 			<li><a href="#recent-work" class="scrolly">Recent Work</a></li>
 			<li><a href="/team" >Team</a></li>
+			<li><a href="/news" >News</a></li>
```
Comment (jamiefolsom) [inline _includes/header.html]: For now, I would comment out the links to this section, so it doesn't appear on the site until we've got some real content in place.

File: _includes/header.html [hunk 2]
```diff
@@ -6,12 +6,14 @@ <h1 id="logo"><a href="/"> <img src="/images/performant.png" height="50px" alt="
...
 			<li><a href="/careers" >Careers</a></li>
       <li><a href="#cta" class="scrolly">Contact Us</a></li>
     {% else %}
 			<li><a href="/#main" >What We Do</a></li>
 			<li><a href="/#recent-work" >Recent Work</a></li>
 			<li><a href="/team" >Team</a></li>
+			<li><a href="/news" >News</a></li>
```
Comment (jamiefolsom) [inline _includes/header.html]: Comment this one out too, for now.

**Comment by jamiefolsom** [comment]
Also, it looks like your git config is not [set up with your github account](https://help.github.com/articles/setting-your-commit-email-address-in-git/). You might want to set that up, so your commits are credited to you.

**Comment by mkobayashi21** [comment]
Rebased, squashed, and renamed changes

**Comment by jamiefolsom** [comment]
Good work @mkobayashi21! Looking better -- here are a couple more ideas.

Formatting the index page: try making it so there's a byline and dateline, next to, or under, the title, on the index page. It's good to use a `<ul>`, as you've done, with each news item being a `<li>`, but you can hide the bullet, and move the date and author so they're adjacent to the title. Currently, that page looks like this:

screenshot 2018-06-08 10 58 17

Not bad, just needs a little cleanup. Also, edit the description of this PR to add "Closes #16" and "Closes #17".

**Comment by mkobayashi21** [comment]
Closes #16 
Closes #17

**Comment by jamiefolsom** [comment]
Hi @mkobayashi21 -- looking good! Move your "Closes" statements to the first comment on this PR, at top, by clicking the pencil button and cutting/pasting. That will close this PR when you merge it. Then this is good to merge (using that process I sent you -- [here](https://gist.github.com/jamiefolsom/084369e96e2451d44751)).

**Comment by jamiefolsom** [comment]
And @mkobayashi21, please rebase and squash commits again before merging.

**Pull request merged**